docs/fullDocDraft.md

```diff
@@ -0,0 +1,361 @@
+# G4CMP Documentation
+
+## Overview
+
+G4CMP is a framework designed to add to the Geant4 toolkit for use in condensed
+matter and low-energy physics. Developed for the low-temperature community,
+G4CMP is capable of modeling several physics processes relevant to phonon and
+charge collection at cryogenic temperatures
+[.](../README.md) These include anisotropic phonon
+transport and focusing, phonon isotope scattering, anharmonic downconversion,
+oblique charge carrier propagation with inter-valley scattering, and emission of
+Neganov-Trofimov-Luke phonons by accelerated carriers
+[.](https://arxiv.org/pdf/1403.4984.pdf)
+
+[//]: # (For second source: sections 2 and 3)
+
+The package provides a collection of particle types, physics processes, and
+supporting utilities for this purpose
+[.](../README.md)
+It has been used by collaborators at the
+SuperCDMS project to successfully reproduce theoretical predictions and
+experimental observations such as phonon caustics, heat pulse propagation times,
+and mean carrier drift velocities[;](https://arxiv.org/pdf/1403.4984.pdf)
+the G4CMP package, however, is sufficiently
+general that it is useful for other experiments employing cryogenic phonon and/
+or ionization detectors[.](https://arxiv.org/pdf/1403.4984.pdf)
+
+[//]: # (For second source: section 4. For last source: section 1)
+
+## [Release Notes](https://github.com/kelseymh/G4CMP/blob/master/ChangeHistory)
```

Comment:
Do we want to use the full URL for these links?  Is that helpful if the documentation is generated and stored elsewhere?

Reply:
I don't think it will so long as the user gets the whole file, or at least all the documentation files. The only reason I left it is because I didn't catch all the URLs that could be turned into relative paths.